src/checkout-buttons/strategies/googlepay/googlepay-button-strategy.ts

```diff
@@ -56,7 +56,7 @@ export default class GooglePayButtonStrategy extends CheckoutButtonStrategy {
     }
 
     private _createSignInButton(containerId: string): HTMLElement {
-        const container = document.querySelector(`#${containerId}`);
+        const container = document.getElementById(`${containerId}`);
```

Comment:
Can you remove the string interpolation? Just `document.getElementById(containerId)` is fine.